src/main/java/racingcar/domain/Position.java

```diff
@@ -0,0 +1,23 @@
+package racingcar.domain;
+
+public class Position {
+    public static final String MOVING_UNIT = "-";
+
+    private String position;
+
+    Position(String position) {
+        this.position = position;
+    }
+
+    public void forward(String movingUnit) {
+        position += movingUnit;
+    }
```

Comment:
```suggestion
    public void forward() {
        position += MOVING_UNIT;
    }
```

movingUnit 이 - 말고 다른 것으로 바뀔수도 있나요?
MOVING_UNIT 을 굳이 public 하게 공개하기 보다는,
캡슐화하는게 요구사항에 더 적합하지 않을까 해서 제안을 해봅니다.